lib/spaceship/portal/portal_client.rb

```diff
@@ -377,6 +377,14 @@ def download_provisioning_profile(profile_id, mac: false)
     end
 
     def delete_provisioning_profile!(profile_id, mac: false)
+      if csrf_tokens.count == 0
+        r = request(:post, "account/#{platform_slug(mac)}/profile/getProvisioningProfile.action", {
```

Comment:
Are you sure it's not enough to just call `provisioning_profiles` here? It's basically doing the same request and we'd save some duplicate code.